test_framework/terraform/aws/centos/main.tf

```diff
@@ -268,4 +284,70 @@ resource "aws_ebs_volume" "lh_aws_hdd_volume" {
   tags = {
     Name = "lh-aws-hdd-volume-${random_string.random_suffix.id}-${count.index}"
   }
+}
+
+# Create load balancer for rancher
+resource "aws_lb_target_group" "lh_aws_lb_tg_443" {
+
+  count = var.create_load_balancer ? 1 : 0
+
+  name     = "lh-aws-lb-tg-443-${random_string.random_suffix.id}"
+  port     = 443
+  protocol = "TCP"
+  vpc_id   = aws_vpc.lh_aws_vpc.id
+  health_check {
+    protocol = "TCP"
+    port = "80"
```

Comment:
Why is this 80? This should be the port opened on each worker node and able to forward to Rancher.

Reply:
It follows the document: https://rancher.com/docs/rancher/v2.6/en/installation/resources/k8s-tutorials/infrastructure-tutorials/nlb/
But other experiments can be done to check the effect on modification of this value.